src/components/customComponents/apiService.js

```diff
@@ -0,0 +1,29 @@
+import axios from 'axios';
+
+const ApiInstance = axios.create({
+    baseURL:'https://wbooks-api-stage.herokuapp.com/api/v1/'
+});
+
+
+const ApiService = {
```

Comment:
create different services for each business unit. here we should have an AuthService and a BookService